.github/workflows/coderabbit-feedback.yml

```
@@ -0,0 +1,15 @@
+on:
+  review_comment:
+    types: [created]
+jobs:
+  add-reaction-encouragement:
+    runs-on: ubuntu-latest
+    if: github.actor == 'coderabbitai[bot]'
+    steps:
+      - uses: actions/github-script@v6
```

actionlint (1.7.9)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)